src/ophys_etl/modules/suite2p_wrapper/utils.py

```diff
@@ -9,7 +9,8 @@ class Suite2PWrapperException(Exception):
 
 def copy_and_add_uid(
```

Comment:
Method name is still copy even though it could also move the file without copying.